CONTRIBUTORS

```diff
@@ -74,6 +74,7 @@ Thomas Stephens <thomas@ustudio.com>
 Tim Plummer <objelisks@google.com>
 Timothy Drews <tdrews@google.com>
 Tomas Bucher <collapsingtesseract@gmail.com>
+Tomas Tichy <mr.tichyt@gmail.com>
```

Comment:
Please add yourself or your company (as appropriate) to AUTHORS as well.  Thanks!